src/anomalib/deploy/inferencers/openvino_inferencer.py

```diff
@@ -247,8 +247,10 @@ def post_process(self, predictions: np.ndarray, metadata: dict | DictConfig | No
         # Common practice in anomaly detection is to assign anomalous
         # label to the prediction if the prediction score is greater
         # than the image threshold.
+        pred_label: str | None = None
         if "image_threshold" in metadata:
-            pred_label = pred_score >= metadata["image_threshold"]
+            pred_idx = pred_score >= metadata["image_threshold"]
+            pred_label = "Anomalous" if pred_idx else "Normal"
```

Comment:
Anomalib is already pre-defined `LabelName` enum for this. Maybe we could use this ?

`from anomalib.data import LabelName`

```suggestion
            pred_label = LabelName.ABNORMAL if pred_idx else LabelName.NORMAL
```

Reply:
Didn't know that, but that seems the correct way of doing it! :-)

Reply:
> We should use the ENUM för src/anomalib/deploy/inferencers/torch_inferencer.py as well.

Yeh, you are right. It would be good to update that one as well